exercises/sum.cpp

```diff
@@ -6,3 +6,16 @@
   Insert the second number: 2
   Sum: 3
 */
+#include <iostream>
+using namespace std;
+int main(){
+  int num1,num2,sum;
+  num1=num2=0;
+  cout<<"Insert the first number: ";
+  cin>>num1;
+  cout<<"Insert the second number: ";
+  cin>>num2;
+  sum=num1+num2;
+  cout<<"Sum:"<<sum<<"\n";
+  return 0;
```

Comment:
metti degli spazi nel codice, `nonscriveretuttoattaccato` se no uno non capirebbe

sposta inoltre il codice in un'altra funzione